Us/06 cadastro equipamentos

Yagoas opened the pull request:
## Modificações
Alterados os status dos equipamentos e a exclusão de equipamentos;
Unidade de cadastro de equipamento padronizada.
## Descrição
Foram alterados os status dos equipamentos no cadastro, tabela de equipamentos e OS e a exclusão de equipamentos aparece apenas para o perfil administrador;
Divisão de Suporte Técnico em Informática foi definida como unidade padrão no cadastro de novos equipamentos / Remoção do campo "Unidade" no cadastro / Refatoração.
## US 06: cadastro equipamentos
## Como foi testado?
Foram testadas em tempo real de acordo com as mudanças no aplicativo

## Protótipo
![image](https://user-images.githubusercontent.com/56437421/213922351-748fa599-e1e9-480b-b2ec-2ee2efeb868e.png)

![image](https://user-images.githubusercontent.com/56437421/213921551-28b0d444-98a6-41f1-8ae0-40bf8dc817df.png)


## Tipo de modificações
- [x] _ ​​correção de bug_
- [x] Nova _feature_

## Lista de controle:
- [x] Meu código segue a folha de estilos implementada
- [x] Meu _commits_ segue a política de commits do repo.

Yagoas committed:
#13 fix: corrigido status e acesso a excluir os equipamentos

Yagoas committed:
#13 fix: corrigido status e acesso a excluir os equipamentos

corrigido status e acesso a excluir os equipamentos

Signed-off-by: Yago Amin Santos <yago.amin@gmail.com>
Co-authored-by: Mateus Fidelis Marinho Maia <200073184@aluno.unb.br>

Yagoas committed:
Merge branch 'us/06-cadastro-equipamentos' of https://github.com/fga-eps-mds/2022-2-Alectrion-FrontEnd into us/06-cadastro-equipamentos

ErickGiffoni commented:
@Yagoas reparei que na _issue_ relativa a essa história nem todos os critérios de aceitação estão marcados... sabe me dizer o motivo ?
Outra coisa, quem são os _assignees_ desse PR ?

Yagoas commented:
> @Yagoas reparei que na _issue_ relativa a essa história nem todos os critérios de aceitação estão marcados... sabe me dizer o motivo ? Outra coisa, quem são os _assignees_ desse PR ?

Não conseguimos finalizar todos os itens do sprint no prazo para enviar na segunda passada, ficou para a sprint seguinte. Assignees foram colocados.

Yagoas committed:
Merge branch 'main' into us/06-cadastro-equipamentos

Yagoas committed:
#13 feature: Unidade de cadastro de equipamento padronizada

Divisão de Suporte Técnico em Informática foi definida como unidade padrão no cadastro de novos equipamentos / Remoção do campo "Unidade" no cadastro / Refatoração

Signed-off-by: Yago Amin Santos <yago.amin@gmail.com>
Co-authored-by: Mateus Fidelis Marinho Maia <200073184@aluno.unb.br>

Yagoas committed:
Merge branch 'dev' into us/06-cadastro-equipamentos

Yagoas committed:
#13 fix: Data de acquisão corrigida

Data de acquisão não estava sendo exibida corretamente / Ajuste da tabela para ficar no dimensionamento correto de acordo com a quantidade de itens no cabeçalho / Refatoração do código

Signed-off-by: Yago Amin Santos <yago.amin@gmail.com>
Co-authored-by: Mateus Fidelis Marinho Maia <200073184@aluno.unb.br>

Yagoas committed:
Update index.tsx

Yagoas committed:
#13 feature: Propriedade do equipamento criada

Propriedade do equipamento "Novo" | "Usado" criada / Refatoração

Signed-off-by: Yago Amin Santos <yago.amin@gmail.com>
Co-authored-by: Mateus Fidelis Marinho Maia <200073184@aluno.unb.br>

Yagoas committed:
Merge branch 'us/06-cadastro-equipamentos' of https://github.com/fga-eps-mds/2022-2-Alectrion-FrontEnd into us/06-cadastro-equipamentos

Yagoas committed:
#13 fix: Altera termos para portugues na visualização

Termos do tipo de equipamento e situação (antigo "status") alterados  / Refatoração

Signed-off-by: Yago Amin Santos <yago.amin@gmail.com>
Co-authored-by: Mateus Fidelis Marinho Maia <200073184@aluno.unb.br>

Yagoas committed:
#13 fix: Correção cadastro Escaneador

Correção do cadastro de equipamento tipo "escaneador" / Ordenação alfabética das propriedades dos equipamentos / Refatoração

Signed-off-by: Yago Amin Santos <yago.amin@gmail.com>
Co-authored-by: Mateus Fidelis Marinho Maia <200073184@aluno.unb.br>

Yagoas committed:
#13 fix: Propriedades dos equipamentos são campos pesquisáveis

Adicionado os campos para busca de todas as propriedades dos equipamentos

Signed-off-by: Yago Amin Santos <yago.amin@gmail.com>
Co-authored-by: Mateus Fidelis Marinho Maia <200073184@aluno.unb.br>

ErickGiffoni commented on src/components/Equipament-Tables/index.tsx:
```diff
@@ -87,10 +90,11 @@ export default function EquipamentsTables({ equipaments }: propType) {
             <StyledTableCell align="center">N° Tombamento </StyledTableCell>
             <StyledTableCell align="center">N° Serie</StyledTableCell>
-            <StyledTableCell align="center">Status</StyledTableCell>
+            <StyledTableCell align="center">Situação</StyledTableCell> {/* renomeamos status para situação */}
```
Legal

ErickGiffoni commented on src/pages/order-service/index.tsx:
```diff
@@ -18,7 +18,8 @@ export type Equipment = {
   serialNumber: string
   type: string
-  status: string
+  situação: string
```
Vou te pedir uma mudança que vai dar um pouco de trabalho, mas quero até a opinião do outros EPS (@JoaoVitorFarias @SamButers @AlineLermen @AlvaroLeles @RuanMoura ) sobre isso:

Quando estiver escrevendo código e for dar nome para variáveis, ou nessas ocasiões de `chave: valor`, não usar acentuações nos nomes das variáveis nem das "chaves".

Então esse código e os outros parecidos nos outros arquivos ficariam assim:
```suggestion
  situacao: string
```

É comum não usarmos acentuações, até porque pode dar erro ou conflito de leitura de arquivo etc a depender do computador, tipo se ele lê em UTF 8 etc.

Yagoas replied:
Alterações feitas.

ErickGiffoni reviewed (changes requested):
Show, boa.
Só mudar a questão colocada no comentário e pedir a revisão de novo que deve ficar ok !

Yagoas committed:
#13 fix: Refatoração

Refatoração de parte do código para seguir os padrões do código fonte

Signed-off-by: Yago Amin Santos <yago.amin@gmail.com>
Co-authored-by: Mateus Fidelis Marinho Maia <200073184@aluno.unb.br>

ErickGiffoni reviewed (approved):
Boa
